openapi.yml

```diff
@@ -1422,13 +1426,14 @@ components:
         - id
         - memberCount
     IdpStatus:
+      type: string
       title: IdpStatus
       x-stoplight:
         id: a3fdr17gch04a
-      type: string
       enum:
         - ARCHIVED
         - ACTIVE
+        - INTERNAL
```

Comment:
@meyerjrr can we expose this as a separate field called "source", rather than adding it to the status field?

the source field would be an enum with "INTERNAL" and "EXTERNAL" being the options.

Reply:
This is to deal with the fact that internal groups can also be archived

Reply:
it would be even better actually if we could have the source be the IDP type, so our sources can be:

- internal
- azure
- google
- onelogin

etc